tests/unit/base/class_list_unittest.cc

```diff
@@ -0,0 +1,26 @@
+/*
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation; either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * Written (W) 2016 Sanuj Sharma
+ */
+
+#include <shogun/base/SGObject.h>
+#include <shogun/base/class_list.h>
+#include <shogun/kernel/GaussianKernel.h>
+#include <gtest/gtest.h>
+
+using namespace shogun;
+
+TEST(class_list, create_gaussian_kernel)
+{
+    const char* class_name = "GaussianKernel";
```

Comment:
@karlnapf @sanuj ok as it works lets remove swig part because we need that mighty list of base classes that should come from other PR. Not sure about all shogun classes.